api/types.go

```diff
@@ -295,10 +295,21 @@ type EmbeddingResponse struct {
 
 // CreateRequest is the request passed to [Client.Create].
 type CreateRequest struct {
-	Model     string `json:"model"`
+	Model    string `json:"model"`
+	Stream   *bool  `json:"stream,omitempty"`
+	Quantize string `json:"quantize,omitempty"`
+
+	From       string           `json:"from,omitempty"`
+	FromModel  *CreateFromModel `json:"from_model,omitempty"`
```

Comment:
Given `from` is from an existing Ollama model, `from_model` seems to conflict with that - would this be better named as `from_files`? or similar

Reply:
This is now changed to `Files` and `CreateFromModel` removed entirely. The server will now detect if you passed it a `safetensors` or `gguf` files.